### ISSUE TITLE
Trying to install this in an existing asp.net MVC site.

### ISSUE THREAD

**Issue by juliandormon** [madskristensen/Miniblog.Core#55]
Sorry. I am a bit of a newbie here.

I have an MVC .net site  running on IIS. I would like to have this run under a directory of that site but I cannot figure out how to install it?

I have set up the blog to run under it's own domain and IIS site (IIS 10) without issue. I would like to however run the site from sub-directory from the other domain.

To that end I have added an application to my other IIS site, named it "blog" and pointed it to the root of the working site. This keeps resulting in a 500 error, but I cannot seem to find what the error says despite enabling detailed errors in the web.config.

Any ides how I may get the blog to run in a sub-directory?

Thanks in advance.

I will continue to try and find the error message.

**Comment by juliandormon** [comment]
I switched the application to a virtual directory and then I can see the following error:
"Log Name:      Application
Source:        IIS AspNetCore Module
Date:          3/7/2018 11:32:36 AM
Event ID:      1000
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      WIN-FMS1UAD9TN6
Description:
Application 'MACHINE/WEBROOT/APPHOST/WWW.DOMAIN.NET' with physical root 'C:\Sites\www.domain.net\' failed to start process with commandline ' ', ErrorCode = '0x80070057 : 0.
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="IIS AspNetCore Module" />
    <EventID Qualifiers="0">1000</EventID>
    <Level>2</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2018-03-07T11:32:36.000000000Z" />
    <EventRecordID>16034273</EventRecordID>
    <Channel>Application</Channel>
    <Computer>WIN-FMS1UAD9TN6</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Application 'MACHINE/WEBROOT/APPHOST/WWW.DOMAIN.NET' with physical root 'C:\Sites\www.domain.net\' failed to start process with commandline ' ', ErrorCode = '0x80070057 : 0.</Data>
  </EventData>
</Event>"

**Comment by juliandormon** [comment]
I added the blog application under the default site in IIS so I could run in it localhost, and still the server returns a 500 response but there is no content. Impossible to see what the error is. There is no error in the Windows Application log, either. I was able to turn on request tracing. That definitely shows that an error occurred but does not provide anything meaningful to help me understand what could be happening.

**Comment by pwdst** [comment]
Is your existing MVC site running .NET Core or .NET Framework? This sounds a lot like an issue with the .NET Core runtime on your computer and/or host. Which version(s) of .NET Core are installed and are you using IIS? In addition to the runtime you will need the [ASP.NET Core Module](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/aspnet-core-module) to run .NET Core apps in a reverse proxy configuration (the as-yet unreleased .NET Core version 2.1 will support running in process).

Verify you have .NET Core 2.0x installed and see https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?tabs=aspnetcore2x and https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/troubleshoot

If this is shared hosting not all hosts support .NET Core - ask your host if they currently support .NET Core 2.0 and if not when (if ever) they plan to do so.

**Comment by juliandormon** [comment]
Thank you for your help. Unfortunately I was not able to get the Minblog.Core started. I did however find your MiniBlog project and I was able to install that as a virtual application under my domain. I had to hide a few of our dependencies from the blog in the web.config file using <location path="." inheritInChildApplications="false">. Thanks so much for your responsiveness and of course for the great work!
